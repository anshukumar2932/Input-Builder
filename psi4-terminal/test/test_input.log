2025-03-22:14:56:21,244 INFO     [psi4.driver.task_planner:285] PLANNING Atomic:  keywords={'function_kwargs': {}}
2025-03-22:14:56:21,250 INFO     [psi4.driver.driver:523] Compute energy(): method=hf, basis=sto-3g, molecule=default, nre=0.7151043387432433
2025-03-22:14:56:21,549 INFO     [psi4.driver.driver:526] Return energy(): -1.1167833178830702
