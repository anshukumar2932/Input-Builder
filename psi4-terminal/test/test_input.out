
    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                               Psi4 1.9.1 release

                         Git: Rev {} zzzzzzz 


    D. G. A. Smith, L. A. Burns, A. C. Simmonett, R. M. Parrish,
    M. C. Schieber, R. Galvelis, P. Kraus, H. Kruse, R. Di Remigio,
    A. Alenaizan, A. M. James, S. Lehtola, J. P. Misiewicz, M. Scheurer,
    R. A. Shaw, J. B. Schriber, Y. Xie, Z. L. Glick, D. A. Sirianni,
    J. S. O'Brien, J. M. Waldrop, A. Kumar, E. G. Hohenstein,
    B. P. Pritchard, B. R. Brooks, H. F. Schaefer III, A. Yu. Sokolov,
    K. Patkowski, A. E. DePrince III, U. Bozkaya, R. A. King,
    F. A. Evangelista, J. M. Turney, T. D. Crawford, C. D. Sherrill,
    J. Chem. Phys. 152(18) 184108 (2020). https://doi.org/10.1063/5.0006002

                            Additional Code Authors
    E. T. Seidl, C. L. Janssen, E. F. Valeev, M. L. Leininger,
    J. F. Gonthier, R. M. Richard, H. R. McAlexander, M. Saitow, X. Wang,
    P. Verma, M. H. Lechner, A. Jiang, S. Behnle, A. G. Heide,
    M. F. Herbst, and D. L. Poole

             Previous Authors, Complete List of Code Contributors,
                       and Citations for Specific Modules
    https://github.com/psi4/psi4/blob/master/codemeta.json
    https://github.com/psi4/psi4/graphs/contributors
    http://psicode.org/psi4manual/master/introduction.html#citing-psifour

    -----------------------------------------------------------------------


    Psi4 started on: Saturday, 22 March 2025 02:56PM

    Process ID: 1
    Host:       3785902fefb8
    PSIDATADIR: /opt/conda/share/psi4
    Memory:     500.0 MiB
    Threads:    1
    
  ==> Input File <==

--------------------------------------------------------------------------
molecule {
    0 1
    H  0.0  0.0  0.0
    H  0.0  0.0  0.74
}

set basis sto-3g
energy('hf')
--------------------------------------------------------------------------

Scratch directory: /tmp/
   => Libint2 <=

    Primary   basis highest AM E, G, H:  5, 4, 3
    Auxiliary basis highest AM E, G, H:  6, 5, 4
    Onebody   basis highest AM E, G, H:  6, 5, 4
    Solid Harmonics ordering:            Gaussian

*** tstart() called on 3785902fefb8
*** at Sat Mar 22 14:56:21 2025

   => Loading Basis Set <=

    Name: STO-3G
    Role: ORBITAL
    Keyword: BASIS
    atoms 1-2 entry H          line    19 file /opt/conda/share/psi4/basis/sto-3g.gbs 


         ---------------------------------------------------------
                                   SCF
               by Justin Turney, Rob Parrish, Andy Simmonett
                          and Daniel G. A. Smith
                              RHF Reference
                        1 Threads,    500 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: d2h
    Full point group: D_inf_h

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
         H            0.000000000000     0.000000000000    -0.370000000000     1.007825032230
         H            0.000000000000     0.000000000000     0.370000000000     1.007825032230

  Running in d2h symmetry.

  Rotational constants: A = ************  B =     61.09110  C =     61.09110 [cm^-1]
  Rotational constants: A = ************  B = 1831464.96157  C = 1831464.96157 [MHz]
  Nuclear repulsion =    0.715104338743244

  Charge       = 0
  Multiplicity = 1
  Electrons    = 2
  Nalpha       = 1
  Nbeta        = 1

  ==> Algorithm <==

  SCF Algorithm Type is DF.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is SAD.
  Energy threshold   = 1.00e-06
  Density threshold  = 1.00e-06
  Integral threshold = 1.00e-12

  ==> Primary Basis <==

  Basis Set: STO-3G
    Blend: STO-3G
    Number of shells: 2
    Number of basis functions: 2
    Number of Cartesian functions: 2
    Spherical Harmonics?: true
    Max angular momentum: 0

   => Loading Basis Set <=

    Name: (STO-3G AUX)
    Role: JKFIT
    Keyword: DF_BASIS_SCF
    atoms 1-2 entry H          line    18 file /opt/conda/share/psi4/basis/def2-universal-jkfit.gbs 

  ==> Integral Setup <==

  DFHelper Memory: AOs need 0.000 GiB; user supplied 0.366 GiB. 
  Using in-core AOs.

  ==> MemDFJK: Density-Fitted J/K Matrices <==

    J tasked:                   Yes
    K tasked:                   Yes
    wK tasked:                   No
    OpenMP threads:               1
    Memory [MiB]:               375
    Algorithm:                 Core
    Schwarz Cutoff:           1E-12
    Mask sparsity (%):       0.0000
    Fitting Condition:        1E-10

   => Auxiliary Basis Set <=

  Basis Set: (STO-3G AUX)
    Blend: DEF2-UNIVERSAL-JKFIT
    Number of shells: 12
    Number of basis functions: 36
    Number of Cartesian functions: 40
    Spherical Harmonics?: true
    Max angular momentum: 2

  Minimum eigenvalue in the overlap matrix is 1.0000000000E+00.
  Reciprocal condition number of the overlap matrix is 1.0000000000E+00.
    Using symmetric orthogonalization.

  ==> Pre-Iterations <==

  SCF Guess: Superposition of Atomic Densities via on-the-fly atomic UHF (no occupation information).

   -------------------------
    Irrep   Nso     Nmo    
   -------------------------
     Ag         1       1 
     B1g        0       0 
     B2g        0       0 
     B3g        0       0 
     Au         0       0 
     B1u        1       1 
     B2u        0       0 
     B3u        0       0 
   -------------------------
    Total       2       2
   -------------------------

  ==> Iterations <==

                           Total Energy        Delta E     RMS |[F,P]|

   @DF-RHF iter SAD:    -0.71843741800502   -7.18437e-01   0.00000e+00 
   @DF-RHF iter   1:    -1.11678331788307   -3.98346e-01   0.00000e+00 DIIS
   @DF-RHF iter   2:    -1.11678331788307    0.00000e+00   0.00000e+00 DIIS
  Energy and wave function converged.


  ==> Post-Iterations <==

    Orbital Energies [Eh]
    ---------------------

    Doubly Occupied:                                                      

       1Ag    -0.578578  

    Virtual:                                                              

       1B1u    0.670950  

    Final Occupation by Irrep:
             Ag   B1g   B2g   B3g    Au   B1u   B2u   B3u 
    DOCC [     1,    0,    0,    0,    0,    0,    0,    0 ]
    NA   [     1,    0,    0,    0,    0,    0,    0,    0 ]
    NB   [     1,    0,    0,    0,    0,    0,    0,    0 ]

  @DF-RHF Final Energy:    -1.11678331788307

   => Energetics <=

    Nuclear Repulsion Energy =              0.7151043387432436
    One-Electron Energy =                  -2.5066195728691336
    Two-Electron Energy =                   0.6747319162428198
    Total Energy =                         -1.1167833178830702

Computation Completed


Properties will be evaluated at   0.000000,   0.000000,   0.000000 [a0]

Properties computed using the SCF density matrix


 Multipole Moments:

 ------------------------------------------------------------------------------------
     Multipole            Electronic (a.u.)      Nuclear  (a.u.)        Total (a.u.)
 ------------------------------------------------------------------------------------

 L = 1.  Multiply by 2.5417464519 to convert [e a0] to [Debye]
 Dipole X            :          0.0000000            0.0000000            0.0000000
 Dipole Y            :          0.0000000            0.0000000            0.0000000
 Dipole Z            :          0.0000000            0.0000000            0.0000000
 Magnitude           :                                                    0.0000000

 ------------------------------------------------------------------------------------

*** tstop() called on 3785902fefb8 at Sat Mar 22 14:56:21 2025
Module time:
	user time   =       0.24 seconds =       0.00 minutes
	system time =       0.02 seconds =       0.00 minutes
	total time  =          0 seconds =       0.00 minutes
Total time:
	user time   =       0.24 seconds =       0.00 minutes
	system time =       0.02 seconds =       0.00 minutes
	total time  =          0 seconds =       0.00 minutes

    Psi4 stopped on: Saturday, 22 March 2025 02:56PM
    Psi4 wall time for execution: 0:00:00.36

*** Psi4 exiting successfully. Buy a developer a beer!
